scripts/js/lib/api/Metadata.ts

```diff
@@ -19,7 +19,8 @@ export type ApiType =
   | "function"
   | "exception"
   | "data"
-  | "struct";
+  | "struct"
+  | "type";
```

Comment:
Ik the code will be a little more complex for `getApiType`, but I think it'd be worth calling this `typedef` rather than `type` to make it clear this is C and not Python types. `type` is very overloaded. I think you only need to update `getApiType` to change its detection, then everything else simply s/type/typedef.